scripts/genunicodetables.lua

```diff
@@ -0,0 +1,227 @@
+#!/usr/bin/lua
```

Comment:
@Pyrohh Other scripts are using `#!/usr/bin/lua` or nothing. In any case they are run by directly calling lua executable and not by calling scripts themselves, so this is irrelevant.